ui/src/components/Map/Map.jsx

```diff
@@ -36,31 +44,50 @@ const Map = () => {
                 stroke="#FFF"
                 geography={geo}
                 fill="#DDD"
+                className="state"
               />
             ))}
             {geographies.map(geo => {
               const centroid = geoCentroid(geo);
               const cur = allStates.find(s => s.val === geo.id);
               return (
+
+
                 <g key={geo.rsmKey + "-name"}>
                   {cur &&
                     centroid[0] > -160 &&
```

Comment:
what do -160 and -67 represent here? if it's just an arbitrary offset, that's alright. If it's an actual value like "Northern Bordern of US" or something, i'd move the value to a const